typescript/cli/src/deploy/utils.ts

```diff
@@ -76,13 +76,17 @@ export async function runPreflightChecksForChains({
   assertSigner(signer);
   logGreen('✅ Signer is valid');
 
-  await assertGasBalances(
+  const lowBalanceExists = await assertGasBalances(
     multiProvider,
     signer,
     chainsToGasCheck ?? chains,
     minGas,
   );
-  logGreen('✅ Balances are sufficient');
+  logGreen(
+    lowBalanceExists
+      ? '⚠️ Balances may not be sufficient'
+      : '✅ Balances are sufficient',
```

Comment:
Is this redundant with the error assertGasBalances already shows?

Reply:
Somewhat, but I kinda prefer it. If you'd like, we could just omit logging anything if insufficient:
```
if (!lowBalanceExists) logGreen( '✅ Balances are sufficient');
```

Reply:
what action would someone take if they see balances may not be sufficient? feels like it would make me think i should do something but not sure what i should do

Reply:
consider fauceting

Reply:
nice callout– we should advise an action. If you take a look at the Drive-by changes section, you can see the current output, which includes:
```
? WARNING: 0x2d54EC30Be0D2583b0d781e10B31905c3c54616d has low balance on alfajores. At least 1.0 recommended but found 0.0 CELO Continue? yes
```

as you've pointed out below, definitely out of scope for this PR, but as a user I'd most enjoy the option to fund my wallet, and re-check balances without having to exit the script (current result of entering `no`)

Reply:
Decided on just logging '✅ Balances are sufficient' when sufficient to keep this change moving